SimPEG/electromagnetics/base_1d.py

```diff
@@ -17,9 +17,66 @@
     validate_integer,
 )
 from .. import props
+from ..maps import Wires
 from empymod.utils import check_hankel
 
-__all__ = ["BaseEM1DSimulation"]
+__all__ = [
+    "BaseEM1DSimulation",
+    "run_em1d_simulation",
+    "OutputType",
+    "ColeColeParameters",
+    "HankelCoefficients",
+]
+
+
+class OutputType(Enum):
+    FORWARD = 1
+    SENSITIVITY = 2
```

Comment:
Don't have to keep this design with Enum, but generally more robust than string comparison. Move the responsibility to the user to provide valid inputs.